File: Assignment1/Leap_year.java
import java.util.*;
public class Leap_year
{
   public static void main(String args[])
{
    Scanner sc=new Scanner(System.in);
int n;
n=sc.nextInt();
if(n%400==0)
  System.out.println("It is a Leap Year");
else if(n%100==0)
    System.out.println("Not a Leap Year");
else
  System.out.println("It is a Leap Year");
   }
}

File: Python/Assignment 1/Leap_year.py

def leap_year(a):
    if a%400==0:
        return True
    elif a%100==0:
        return False
    elif a%4==0:
        return True
    else:
        return False      
year=int(input("Enter the year"))
print(year,"is a Leap year ?",leap_year(year))